avm/res/insights/metric-alert/main.bicep

```diff
@@ -65,15 +65,7 @@ param autoMitigate bool = true
 param actions array = []
 
 @description('Optional. Maps to the \'odata.type\' field. Specifies the type of the alert criteria.')
```

Comment:
Required or optional? The UDT is currently required - hence you must provide it in the defaults test. If correct, then the description should be updated.